### ISSUE TITLE
Trimming Multiple Copies of same adapter.

### ISSUE THREAD

**Issue by PaulAONeill** [OpenGene/fastp#233]
Hello,

I found a problem sample which has multiple copies of a single adaptor in the same read.
fastp identifies the adaptor correctly `AGATCGGAAGAGCACACGTCTGAACTCCAGTCA`

and removed it from the sequence
`AGATCGGAAGAGCACACGAGATCGGAAGAGCACACGTCTGAACTCCAGTCA`

leaving
`AGATCGGAAGAGCACACG`

I can't see a parameter to remove adapters multiple times - could you please advise.

Thanks

**Comment by yech1990** [comment]
use the `-n` parameter in `cutadapt`...

Seems that `fastp` is not under maintenance now.

**Comment by sfchen** [comment]
> use the `-n` parameter in `cutadapt`...
> 
> Seems that `fastp` is not under maintenance now.

No, I was too busy on anti-COVID19 activities. But I am still maintaining fastp
